programmatic-upload-aws-cli.md

```diff
@@ -26,8 +26,24 @@ The DCC will create buckets and allow centers access to use the AWS CLI to uploa
 ![bucket_sync_diagram](https://user-images.githubusercontent.com/63608514/93384855-375e8180-f81a-11ea-91d7-400bb7ffa8a8.jpeg)
 
 
+
+
 ### Examples:
 
+**Direct-to-bucket S3 upload**:
```

Comment:
Maybe we can move this section below the 'cp' and 'put-object' examples since it makes use of the `aws s3 cp` command? And this subject+description may be a bit more descriptive:

**Shell script using `aws s3 cp` command**
The example shell script below copies all files in a local directory to a specified folder in an S3 bucket with the required flags. It can be modified for more complex folder structures.